org.faktorips.valuetypes.joda/pom.xml

```diff
@@ -1,37 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
-    xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<project
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>org.faktorips.valuetypes.joda</artifactId>
-    <packaging>eclipse-plugin</packaging>
-
     <parent>
         <groupId>org.faktorips</groupId>
-        <artifactId>org.faktorips.aggregator.plugins</artifactId>
-        <version>22.6.0-SNAPSHOT</version>
-        <relativePath>../org.faktorips.aggregator/plugins/pom.xml</relativePath>
+        <artifactId>base</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
```

Comment:
Das hatte ich in den vorherigen Commits übersehen; warum hat base die Version 1.0.0 und nicht 22.6.0 wie alle anderen POMs?

Reply:
Hat keinen Grund, habe ich angepasst.